altair/vegalite/v5/api.py

```diff
@@ -107,6 +107,8 @@ def _prepare_data(data, context=None):
 # ------------------------------------------------------------------------
 # Aliases & specializations
 Bin = core.BinParams
+Impute = core.ImputeParams
+Title = core.TitleParams
```

Comment:
```suggestion
Title = core.TitleParams
Stack = core.StackOffset
```

Should we also add this one? That seems to be all of them then